pkg/canary/comparator/comparator.go

```diff
@@ -285,6 +289,11 @@ func (c *Comparator) run() {
 				go c.metricTest(time.Now())
 			}
 			c.metTestMtx.Unlock()
+			if firstMt {
+				// After the first tick which is at a random time, resume at the normal periodic interval
+				firstMt = false
+				mt.Reset(c.metricTestInterval)
+			}
```

Comment:
I wonder whether a sleep would be easier to understand.

Reply:
a sleep would be more complicated to implement without affecting the other tickers in the select. We don't want to interfere with the spot check test for example, and make it wait up to 1hr before running.

Reply:
Ok. Then let's go with this solution :slightly_smiling_face: